tests/components/test_nlp.py

```diff
@@ -617,7 +617,7 @@ def test_sys_entity_feature(kwik_e_mart_nlp):
 
 
 test_data_dyn = [
-    ("kadubeesanahalli", None, "store_info", "", ""),
+    ("kadubeesanahalli", None, "store_info", "get_store_hours", ""),
```

Comment:
This test sometime fails when "kadubeesanahalli" is classified as "get_store_hours", so I am pinning a more narrow scope here